android/BUILD.gn

```diff
@@ -26,5 +27,6 @@ java_cpp_template("brave_config_java") {
     "BRAVE_ANDROID_P3A_ENABLED=$brave_p3a_enabled",
     "BRAVE_ANDROID_WEBCOMPAT_REPORT_ENDPOINT=\"$webcompat_report_api_endpoint\"",
     "BRAVE_ANDROID_AI_CHAT_ENABLED=$enable_ai_chat",
+    "BRAVE_ANDROID_FEEDBACK_REPORT_ENDPOINT=\"$android_feedback_report_api_endpoint\"",
```

Comment:
I believe `android_feedback_report_api_endpoint` should be `brave_android_feedback_report_api_endpoint` like the other Android ones.